src/zgossip.c

```diff
@@ -563,7 +563,6 @@ monitor_handler (zloop_t *loop, zsock_t *monitor_socket, void *argument)
     if (monitor) {
         zmsg_t *msg = zmsg_recv (monitor);
         if (msg) {
-            zmsg_print (msg);
```

Comment:
Why is it useless? Wouldn't it be better to hide it behind the verbose flag?

Reply:
zgossip does not publicly support the VERBOSE command yet. I have commented some other code for the day it will support it. What makes things complicated is that zgossip relies on an automatically generated engine (zgossip_engine.inc) which supports verbose in its private structure but does not expose it.